### PR TITLE
specified role assignment

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1763

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @mypil, for your information

Thank you

File: microsoft-365/compliance/data-loss-prevention-policies.md [hunk 1]
```diff
@@ -501,8 +501,10 @@ Members of your compliance team who will create DLP policies need permissions to
 1. Create a group in Microsoft 365 and add compliance officers to it.
     
 2. Create a role group on the **Permissions** page of the Security &amp; Compliance Center. 
-    
-3. Add the Microsoft 365 group to the role group.
+
+3. Assign **DLP Compliance Management** role to the role group
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-loss-prevention-policies.md]:
```suggestion
3. Assign a **DLP Compliance Management** role to the role group.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by UnsnappedKhan8** [comment]
Salt

**Comment by UnsnappedKhan8** [comment]
Global Admin

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. This is based on issue #1763

Thank you.

cc: @kenwith

**Comment by PrasadKasireddy** [comment]
These changes are updated and including the additional information in PR https://github.com/MicrosoftDocs/microsoft-365-docs/pull/1891

**Pull request merged**